src/main/resources/templates/macros/report/embeddings.vm

```diff
@@ -31,6 +33,10 @@
   #includeExpandableEmbedding($embedding, $image_type, "<img src=${Quote}data:image/$image_type;base64,$embedding.getData()${Quote}>", $index)
 #end
 
+#macro(includeImageReferenceEmbedding, $embedding, $index)
+  #set($Quote = '"')
+  #includeExpandableEmbedding($embedding, "Image", "<img src=${Quote}$embedding.getData()${Quote}>", $index)
```

Comment:
@k-g-aleksandrov 
I found that here it should be getDecodedData() instead of getData()

let me know you have any objective for this change